src/content/docs/synthetics/synthetic-monitoring/private-locations/containerized-private-minion-cpm-configuration.mdx

```diff
@@ -12,6 +12,11 @@ redirects:
 freshnessValidatedDate: never
 ---
 
+<Callout variant="important">
+  The containerized private minion (CPM) is [end of life on October 22, 2024](/whats-new/2024/04/whats-new-04-09-eol-synthetics-runtime-cpm/). 
```

Comment:
```suggestion
  On October 22, 2024, we will [end of life](/whats-new/2024/04/whats-new-04-09-eol-synthetics-runtime-cpm) (EOL) the containerized private minion (CPM). You need to update to the [newest runtime](/docs/synthetics/synthetic-monitoring/using-monitors/runtime-upgrade-ui), then transition your CPMs to the [synthetics job manager](/docs/synthetics/synthetic-monitoring/private-locations/job-manager-transition-guide) to continue monitoring your private locations. 
```

Reply:
@bpeckNR How about this? Not sure if by transitioning to the job manager, it "bundles up" the update to the newest runtime, or if it's a "do X, then do Y." Either way, we should make those steps clear since this is a breaking change

Reply:
There are multiple steps required if the customer is using the CPM. It requires installing job manager then updating to the newest runtime. 

The recommended path is for the customer to:

1. Install Job Manager using the same private location key as their CPM installation
2. Upgrade the runtime in the monitors that are assigned to that private location
3. Once no jobs are running on the CPM, uninstall the CPM